client/Containers/Cart/ProductCard.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  ProductCardForCart,
+  Column,
+} from '../../components/styles/ProductCardForCart';
+import DeleteButton from '../../components/styles/DeleteButton';
+import IncrementButton from '../../components/styles/IncrementButton';
+import { updateCartProduct } from '../../store/cart/cart';
+
+const ProductCard = ({ cartProduct, updateCartProduct }) => {
+  console.log(cartProduct);
+  const { quantity, product, productId } = cartProduct;
+  const { price, name, imageUrl } = product;
+  const totalPriceOfItem = quantity * price;
```

Comment:
Doing some destructuring to avoid a lot of dot notation. I also find that destructuring can help with understanding the structure of an object you're using, whereas dot notation isn't as clear.